app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -62,6 +62,9 @@ interface ICloneGithubRepositoryProps {
    * available for cloning.
    */
   readonly onRefreshRepositories: (account: Account) => void
+
+  /** Initiate cloning if a repository and cloning path are selected and no current errors. */
+  readonly cloneIfCloningEnabled?: () => void
```

Comment:
```suggestion
  readonly cloneIfCloningEnabled: () => void
```

This can be made non-optional since this component is only used in 1 place.

Reply:
Excellent, thanks for the feedback! Since this no longer needs to be optional I went ahead and cleaned this up a bit to match the other prop function calls.